examples/advanced/job_api/pt/fedavg_script_executor_cifar10.py

```diff
@@ -14,34 +14,23 @@
 
 from src.net import Net
 
-from nvflare import FedAvg, FedJob, ScriptExecutor
+from nvflare.app_common.executors.script_executor import ScriptExecutor
+from nvflare.job_config.pt.fed_avg import FedAvgJob
 
 if __name__ == "__main__":
     n_clients = 2
     num_rounds = 2
     train_script = "src/cifar10_fl.py"
 
-    job = FedJob(name="cifar10_fedavg")
-
-    # Define the controller workflow and send to server
-    controller = FedAvg(
-        num_clients=n_clients,
-        num_rounds=num_rounds,
-    )
-    job.to(controller, "server")
-    # job.to_server(controller)
-
-    # Define the initial global model and send to server
-    job.to(Net(), "server")
-    # job.to_server(Net())
+    job = FedAvgJob(name="cifar10_fedavg", num_rounds=num_rounds, n_clients=n_clients, initial_model=Net())
```

Comment:
This object is similar to our job_templates concept, which is good, as it save users some time if they want to use these.